src/interface/src/app/services/plan.service.ts

```diff
@@ -232,27 +234,29 @@ export class PlanService {
       );
   }
 
-  /** Creates multiple project areas for a project. */
-  bulkCreateProjectAreas(projectId: number, projectAreas: GeoJSON.GeoJSON[]) {
-    const url = BackendConstants.END_POINT.concat(
-      '/plan/create_project_areas_for_project/'
-    );
-    return this.http
-      .post<number>(
-        url,
-        {
-          project_id: Number(projectId),
-          geometries: projectAreas,
-        },
-        {
-          withCredentials: true,
-        }
-      )
-      .pipe(
-        take(1),
-        map(() => null)
-      );
-  }
+  // Currently unused and unimplemented
+  // TODO either adapt to use scenarioID or remove
+  // /** Creates multiple project areas for a project. */
+  // bulkCreateProjectAreas(projectId: number, projectAreas: GeoJSON.GeoJSON[]) {
+  //   const url = BackendConstants.END_POINT.concat(
+  //     '/plan/create_project_areas_for_project/'
```

Comment:
This is using the old "plan/xyz" endpoints, so I rather remove this completely and re implement it, given that we want to move to the newer `plannning/xyz` endpoints

Reply:
Cool, I'll go ahead and delete all the methods that aren't currently implemented and add a todo summarizing which ones need to be potentially re-implemented

Reply:
Nevermind, that opens up a whole other can of worms 😅  — deleted this one with a todo to reimplement and added todo's to delete the rest in a future pr